addon/components/polaris-banner.hbs

```diff
@@ -1,14 +1,16 @@
 <div
   class={{this.cssClasses}}
   tabindex="0"
-  aria-live="polite"
+  role={{this.useBannerAttributes.ariaRoleType}}
+  aria-live={{if this.stopAnnouncements "off" "polite"}}
   aria-describedby={{this.contentId}}
   aria-labelledby={{this.headingId}}
-  role={{this.role}}
   {{on "mouseup" this.handleMouseUpByBlurring}}
```

Comment:
I think we need to tweak this action for `moseup` event to also set shouldShowFocus to `false` ([source](https://github.com/Shopify/polaris-react/blob/9a167d3565573739b4f2dfa8e516bc8a13a0f179/src/components/Banner/Banner.tsx#L269))